src/utils/ucc_coll_utils.c

```diff
@@ -229,6 +229,47 @@ ucc_ep_map_t ucc_ep_map_from_array(ucc_rank_t **array, ucc_rank_t size,
     return map;
 }
 
+ucc_ep_map_t ucc_ep_map_from_array_64(uint64_t **array, ucc_rank_t size,
```

Comment:
this fn 99% code duplication of ucc_ep_map_from_array. Lets unify them. Add static ucc_ep_map_from_array_generic that takes void *array and size_t elem_size. Then
ucc_ep_map_from_array returns ucc_ep_map_from_array_generic(array, 4, ...), and the new one will call same with (array, 8, ...)

Reply:
Added generic function